src/targets/services/konnectorAlerts.spec.js

```diff
@@ -16,6 +16,18 @@ jest.mock('cozy-ui/transpiled/react/AppLinker', () => ({
   generateUniversalLink: () => 'universal-link'
 }))
 
+// String.prototype.matchAll only available for node > 12
+const matchAll = (str, regex) => {
+  // Need to have the "g" flag
+  const gRegex = new RegExp(regex, 'g')
+  let matches
+  const res = []
+  while ((matches = gRegex.exec(str)) !== null) {
+    res.push(matches[0])
+  }
+  return res
+}
+
```

Comment:
nit : Je suppose que tu as copié/collé la fonction tel quel. Question innocente : est-ce qu'il n'y a pas moyen de plutôt l'importer ?

Reply:
I've extracted the function to utils/matchAll, thanks.